### DISCUSSION TITLE
[Feature request] CLIP interrogator tab (load image, get prompt ideas.)

### DISCUSSION THREAD

**Discussion by Nesuhst** [hlky/stable-diffusion-webui#325] [Feature suggestions]
"The CLIP Interrogator uses the OpenAI CLIP models to test a given image against a variety of artists, mediums, and styles to study how the different models see the content of the image. It also combines the results with BLIP caption to suggest a text prompt to create more images similar to what was given."

https://github.com/pharmapsychotic/clip-interrogator

**Comment by altryne** [comment]
This is one of the first things I learned gradio for! 
@pharmapsychotic 's  repo and Colab are amazing. I think we'd need to think about the scope of the webui though. 

But a good idea to mull around
